WeatherApp/Services/WeatherApiService.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  WeatherApiService.swift
+//  WeatherApp
+//
+//  Created by Lucija Balja on 03/08/2020.
+//  Copyright © 2020 Lucija Balja. All rights reserved.
+//
+
+import Foundation
+
+class WeatherApiService {
+    let apiURL = "https://api.openweathermap.org/data/2.5/weather?appid=\(Global.apiKey)&units=metric"
+    
+    func fetchWeather(for city: String, completion: @escaping (WeatherData) -> Void) {
```

Comment:
Note that completion does not provide a way to signal it that error has occurred. You do not need to fix it now but think about a way to do it :)